python/lsst/ip/diffim/subtractImages.py

```diff
@@ -390,7 +390,7 @@ def run(self, template, science, sources, finalizedPsfApCorrCatalog=None,
         try:
```

Comment:
Sure, more explanation is helpful. If you are adding that comment, also mention that we catch `InvalidParameterError` in case the point where we are evaluating the PSF lands in a gap in the template.